app/views/posts/index.html.erb

```diff
@@ -1 +1,35 @@
-<h1> Here is a list of posts for a given user </h1>
\ No newline at end of file
+<%= render "shared/user", user: @user %>
+
+<% if @posts.any? %>
+  <% @posts.each_with_index do |post, index| %>
+    <a href="/users/<%= post.author_id %>/posts/<%= post.id %>?index=<%= index + 1 %>" class="text-decoration-none">
+      <%= render "shared/post", post: post, post_index: index + 1 %>
+      <div class="row justify-content-center align-items-center my-1">
+         <div class="col-lg-8 col-md-11 col-11 border p-2 ">
+            <% if post.comments.any? %>
+               <% post.comments.first(3).each do |comment| %>
+                  <p><%= comment.user.name %>: <%= comment.text %></p>
+               <% end %>
```

Comment:
- [ ] Same here, As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Use%20methods%20that%20you%20have%20created%20in%20the%20Blog%20App%20%2D%20processing%20data%20in%20models.), Please use the `recent_comments` method to render the 5 most recent comments only for the post.
```suggestion
               <% post.recent_comments.each do |comment| %>
                  <p><%= comment.user.name %>: <%= comment.text %></p>
               <% end %>
```